### ISSUE TITLE
Read the following topics

### ISSUE THREAD

**Issue by pruthvi2103** [pruthvi2103/skincancerdetect-model#2]
- CNN,
- Activation functions
- keras
- batch normalization 
- flattening
- sigmoid function
